scripts/src/java/org/oppia/android/scripts/ScriptResultConstants.kt

```diff
@@ -0,0 +1,7 @@
+package org.oppia.android.scripts
+
+object ScriptResultConstants {
+  /** RegexPatternValidationCheck output message constants **/
+  val REGEX_CHECKS_PASSED: String = "REGEX PATTERN CHECKS PASSED"
+  val REGEX_CHECKS_FAILED: String = "REGEX PATTERN CHECKS FAILED"
```

Comment:
Why do we need these?

Reply:
This was based on [this](https://github.com/oppia/oppia-android/pull/3340#discussion_r654961042) comment. It also helps in asserting the check passed/failed message string in the tests.

Reply:
@BenHenning 
Done.